tools/depends/configure.ac

```diff
@@ -458,10 +452,12 @@ case $use_platform in
      platform_ldflags="-lpthread"
      ;;
   tvos)
-    if test "$platform_os" != "darwin_embedded"; then
-      AC_MSG_ERROR([$use_platform is only supported on the darwin_embedded platform])
-    fi
-    ;;
+     platform_cflags+=" -fembed-bitcode"
+     platform_cxxflags+=" -fembed-bitcode"
```

Comment:
Why add adding the `-fembed-bitcode` flags here?

Reply:
only real reason was trying to split out the platform specific stuff. Started wondering if it was worth adding platform=ios, but never went that route in the end.
Currently ios doesnt use the -fembed-bitcode, but eventually it may i guess.